pkg/operator/starter.go

```diff
@@ -64,18 +75,28 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		Client:    operatorConfigClient.OperatorV1(),
 	}
 
+	versionGetter := status.NewVersionGetter()
+	clusterOperator, err := configClient.ConfigV1().ClusterOperators().Get(clusterOperatorName, metav1.GetOptions{})
+	if err != nil && !errors.IsNotFound(err) {
+		return err
+	}
+
+	for _, version := range clusterOperator.Status.Versions {
+		versionGetter.SetVersion(version.Name, version.Version)
+	}
+	versionGetter.SetVersion("operator", os.Getenv("OPERATOR_IMAGE_VERSION"))
```

Comment:
I know I saw this code in a different repo, too, but won't this just flip the version of `operator` before we perform the checks that we should be updating it?

Reply:
yes, it's in cluster-kube-apiserver, pkg/operator/starter.go.  I agree.  I was confused, meant to ask about that, I'll ask them, leave it out for now.  (I added that yesterday late and forgot about it, thanks)